assets/js/helpers/boot.js

```diff
@@ -74,6 +74,24 @@ $(document).ready(function() {
           }
     });
   });
+  $.getScript("https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js",function(){
+    window.bootTopic();
+    $.getScript("https://cdn.mathjax.org/mathjax/latest/MathJax.js",function(){
+      MathJax.Hub.Config({
+          jax: ["input/TeX","input/MathML","input/AsciiMath","output/CommonHTML"],
+          extensions: ["tex2jax.js","mml2jax.js","asciimath2jax.js","MathMenu.js","MathZoom.js","AssistiveMML.js"],
+          TeX: {
+            extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
+          },
+          tex2jax: {
+            skipTags:["code"]
+          }
+        });
+      MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
+      $.getScript("/js/dependencies/async_loader.js",function(){
```

Comment:
Let's try with `$(window).on("load", handler)`
More generally the js is becoming quite ugly, I propose that we refactor it in this fashion:
- One boot.js file that will:
    - define a `window.boot` function. This function will just call other sub `boot` function like `bootTopic`
    - contains $(window).on("load", window.boot)
-  In every file where there is $(document).ready,
    - replace the content with a function like `bootTableFiltering` or `bootToggle`, ...
    - put it in the `boot` function in `boot.js`
- Call boot in  `rerenderBody` in async_loader instead of doing all manually